### PR TITLE
Added text changes to privacy statement, email page

### PR DESCRIPTION
**Pull request by Maxim-Lam**
### Description

List of proposed changes:

Added finalized privacy statement

added "or" at the end of a bullet list item for consistency

Separated email tooltip into two paragraphs

### What to test for/How to test

on expectations page, notice new privacy statement with links.

on expectations page, where it says "in person OR by mail within Canada OR"

on email page, tooltip should now be separated into two paragraphs from "for child applications"

### CODE REVIEW

File: public/locales/en/expectations.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
   "header-purpose": "Check the status of your passport application",
   "header-avoid-waiting": "Avoid waiting on the phone and request the status of your application online",
   "header-who-can-check": "Who can check their status online",
-  "header-privacy": "Privacy",
+  "header-privacy": "Privacy Notice Statement",
```
Comment (gregory-j-baker) [inline public/locales/en/expectations.json:5]: This should be sentence-cased.
See: <https://www.canada.ca/en/treasury-board-secretariat/services/government-communications/canada-content-style-guide.html#wp4-1-1>

File: public/locales/fr/expectations.json [hunk 2]
```diff
@@ -29,7 +29,17 @@
...
     }
   },
-  "description-privacy": "Avez-vous lu et accepté les conditions d'utilisation et de confidentialité?",
+  "description-privacy": {
+    "1": "Les renseignements fournis sont recueillis en vertu de l'article 5.1(1)(a)(i) de la Loi sur le ministère de l'Emploi et du Développement social et/ou de l'article 12(e) du Décret sur les passeports canadiens dans le but de vous fournir l'état de votre demande de passeport. Si vous refusez de fournir des renseignements, vous ne serez pas en mesure de consulter l'état de votre demande de passeport.",
+    "2": "De plus, nous ne recueillons votre adresse courriel que si vous demandez à Emploi et Développement social Canada (EDSC)C d'envoyer votre numéro de dossier de demande de passeport par courriel. Sans votre adresse courriel, EDSC ne peut pas envoyer votre numéro de demande de passeport par voie électronique.",
+    "3": "Your personal information may be shared within Service Canada and with Immigration, Refugees and Citizenship Canada to improve services through evaluations, reports and research. This will never result in an administrative decision being made about you.",
```
Comment (gregory-j-baker) [inline public/locales/fr/expectations.json:35]: Should this be French?

File: public/locales/fr/expectations.json [hunk 3]
```diff
@@ -29,7 +29,17 @@
...
+    "4-link": "https://www.microsoft.com/fr-ca/trust-center/privacy",
+    "5": "Vos renseignements personnels sont administrés conformément à la Loi sur le ministère de l'Emploi et du Développement social, la Loi sur la protection des renseignements personnels et les autres lois applicables. Vous avez le droit d'accéder à vos renseignements personnels et de les protéger, tel que décrit dans le fichier de renseignements personnels - Programme de passeport (EDSC PPU 708). Les instructions pour obtenir ces informations sont décrites dans la publication gouvernementale intitulée <2>Renseignements sur les programmes et les fonds de renseignements</2>.",
+    "5-link": "https://www.canada.ca/fr/emploi-developpement-social/ministere/transparence/aai/rapports/infosource.html",
+    "6": "Vous avez le droit de déposer une plainte auprès du Commissariat à la protection de la vie privée du Canada au sujet du traitement de vos renseignements personnels par EDSC: <2>https://www.priv.gc.ca/fr/signaler-un-probleme</2>.",
```
Comment (gregory-j-baker) [inline public/locales/fr/expectations.json:40]:
```suggestion
    "6": "Vous avez le droit de déposer une plainte auprès du Commissariat à la protection de la vie privée du Canada au sujet du traitement de vos renseignements personnels par EDSC\u00a0: <2>https://www.priv.gc.ca/fr/signaler-un-probleme</2>.",
```

Comment (gregory-j-baker) [reply public/locales/fr/expectations.json:40]: I'm unsure about this one since I don't know the nuances of French punctuation.

**Review by gregory-j-baker — CHANGES_REQUESTED** [review]

File: public/locales/en/expectations.json [hunk 4]
```diff
@@ -29,7 +29,17 @@
...
+  "description-privacy": {
+    "1": "The information provided is collected under section 5.1(1)(a)(i) of the Department of Employment and Social Development Act and/or section 12(e) of the Canadian Passport Order for the purpose of providing you with the status of your passport application. Refusing to provide information will result in not being able to view the status of your passport application.",
+    "2": "In addition, we collect your email address only if you require Employment and Social Development Canada (ESDC) to send your passport application file number by email. Without your email address, ESDC cannot send your passport application number electronically.",
+    "3": "Your personal information may be shared within Service Canada and with Immigration, Refugees and Citizenship Canada to improve services through evaluations, reports and research. This will never result in an administrative decision being made about you.",
+    "4": "ESDC carries out this status checker application using services from third-party provider Microsoft Corporation, which may be subject to foreign laws. For more information, see <2>Microsoft Data Privacy Policy & Privacy Principles</2>.",
```
Comment (sebastien-comeau) [inline public/locales/en/expectations.json:36]: Change `<2>Microsoft Data Privacy Policy & Privacy Principles</2>` to `<Link>Microsoft Data Privacy Policy & Privacy Principles</Link>` and use `Trans` property `components` like this:

```tsx
<Trans
  i18nKey={'expectations:description-privacy.4'}
  components={{ Link: <ExternalLink href={t('description-privacy.4-link')} /> }}
/>
```

https://react.i18next.com/latest/trans-component#alternative-usage-which-lists-the-components-v11.6.0

Do this of the other strings.

File: public/locales/fr/expectations.json (hunk 3 above)
Comment (sebastien-comeau) [inline public/locales/fr/expectations.json:40]: Add `\u00a0` before the colon in `EDSC:`

File: pages/expectations.tsx [hunk 5]
```diff
@@ -53,7 +54,54 @@ const Expectations: FC = () => {
         <p className="m-0">{t('do-not-travel')}</p>
       </blockquote>
       <h2 className="h2">{t('header-privacy')}</h2>
-      <p>{t('description-privacy')}</p>
+      <p>
+        <Trans i18nKey={'expectations:description-privacy.1'}>
```
Comment (sebastien-comeau) [inline pages/expectations.tsx:58]: No need to use `Trans` component with `description-privacy` keys 1, 2 and 3.

File: pages/expectations.tsx [hunk 6]
```diff
@@ -53,7 +54,54 @@ const Expectations: FC = () => {
...
+      <p>
+        <Trans i18nKey={'expectations:description-privacy.3'}>
+          Insert feedback{' '}
+          <ExternalLink href={t('expectations:description-privacy.3.3-link')}>
+            Link
+          </ExternalLink>
+        </Trans>
+      </p>
+      <p>
+        <Trans i18nKey={'expectations:description-privacy.4'}>
```
Comment (sebastien-comeau) [inline pages/expectations.tsx:82]: Use `Trans` component property `components` as mentionned in a previous comment for keys 4, 5 and 6.

Link key `description-privacy.4.4-link` is wrong. It should be `description-privacy.4-link`. Same for the other keys.

File: public/locales/fr/expectations.json (hunk 3 above)
Comment (gregory-j-baker) [reply public/locales/fr/expectations.json:40]: https://github.com/DTS-STN/passport-status/pull/240#discussion_r1071492001

File: public/locales/fr/expectations.json (hunk 2 above)
Comment (Maxim-Lam) [reply public/locales/fr/expectations.json:35]: totally lol, missed a copy paste..

File: public/locales/en/expectations.json (hunk 4 above)
Comment (Maxim-Lam) [reply public/locales/en/expectations.json]: A child element is needed for ExternalLink so a dummy "link" was added

**Review by gregory-j-baker — APPROVED** [review]

**Review by sebastien-comeau — APPROVED** [review]

**Pull request merged**